x-pack/plugin/core/src/test/java/org/elasticsearch/license/XPackLicenseStateTests.java

```diff
@@ -196,6 +204,8 @@ public void testSecurityGold() {
         assertThat(licenseState.isDocumentAndFieldLevelSecurityAllowed(), is(false));
         assertThat(licenseState.allowedRealmType(), is(XPackLicenseState.AllowedRealmType.DEFAULT));
         assertThat(licenseState.isCustomRoleProvidersAllowed(), is(false));
+        assertThat(licenseState.isTokenServiceAllowed(), is(true));
```

Comment:
Shouldn't TokenService be disallowed in standard?

Reply:
This is a test for Gold.

Reply:
True, I was confused in the split view in :octocat: UI

Reply:
UX problems in :octocat: ? Who would have thunked it?